templates/aakbcms/docker-compose.yml

```diff
@@ -30,7 +30,7 @@ services:
       - PHP_XDEBUG_REMOTE_CONNECT_BACK=${PHP_XDEBUG_REMOTE_CONNECT_BACK:-0}
       - PHP_MAX_EXECUTION_TIME=300
       - PHP_MEMORY_LIMIT=512M
-      # - PHP_MAIL=1 # Uncomment to enable mailhog.
+      - PHP_MAIL=1 # Uncomment to enable mailhog.
```

Comment:
I suggest we remove `# Uncomment to enable mailhog.` since it no longer adds any useful information.